_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -382,6 +384,27 @@ public static function register_endpoints() {
 			'permission_callback' => __CLASS__ . '::activate_plugins_permission_check',
 		) );
 
+		// Publicize: get connection list data for current user.
+		register_rest_route( 'jetpack/v4', '/publicize/connections', array(
```

Comment:
I'm not sure how we can have the `/jetpack/v4` endpoints accessible in Jetpack, Calypso and WP.com at the same time (also mentioned here https://github.com/Automattic/wp-calypso/pull/28102#pullrequestreview-168743801). Perhaps @mdawaffe will have some suggestions.

Reply:
Yeah, fair point. I was striving for consistency with existing endpoints in Jetpack's `class.core-rest-api-endpoints.php` (which are all in `/jetpack/v4`). I think this is part of a bigger question how all of those endpoints are going to work.

Reply:
I don't think we have a firm decision here.

My opinion:
* `jetpack/v4` should be for things dealing with the settings for the Jetpack plugin itself: activate modules, disabling the connection, etc. These endpoints are available to the Jetpack dashboard (and to Calypso via some older forwarding). These endpoints don't need to be available for WordPress.com sites, since it doesn't make sense there (at least for now).
* `wpcom/v2` should be things dealing with the settings for the features in Jetpack: liking a post, creating a contact form, generating a post by email address, etc. These endpoints are available for both Jetpack sites and WordPress.com sites, since they make sense in both contexts.

According to that opinion, these Publicize endpoints would fit best under `wpcom/v2`.